content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -0,0 +1,593 @@
+
+```js
+// Continues from previous examples.
+// earliestLedgerVersion was noted when the transaction was submitted.
+// txID was noted when the transaction was signed.
+tx = await api.getTransaction(txID, {minLedgerVersion: earliestLedgerVersion})
+console.log("Transaction result:", tx.outcome.result)
+console.log("Balance changes:", JSON.stringify(tx.outcome.balanceChanges))
+```
+
+The RippleAPI `getTransaction()` method only returns success if the transaction is in a validated ledger version.
+
+**Caution:** Other APIs may return tentative results from ledger versions that have not yet been validated. For example, if you use the `rippled` APIs' [tx method][], be sure to look for `"validated": true` in the response to confirm that the data comes from a validated ledger version. Transaction results that are not from a validated ledger version are subject to change. For more information, see [Finality of Results](finality-of-results.html).
+
+<div class="interactive-block" id="interactive-check">
+<div class="interactive-block-inner">
+  <div class="breadcrumbs-wrap">
+    <ul class="breadcrumb prereqs">
+      <li class="breadcrumb-item active bc-generate"><a href="#interactive-generate">Generate</a></li>
+      <li class="breadcrumb-item disabled bc-connect"><a href="#interactive-connect">Connect</a></li>
+      <li class="breadcrumb-item disabled bc-prepare"><a href="#interactive-prepare">Prepare</a></li>
+      <li class="breadcrumb-item disabled bc-sign"><a href="#interactive-sign">Sign</a></li>
+      <li class="breadcrumb-item disabled bc-submit"><a href="#interactive-submit">Submit</a></li>
+      <li class="breadcrumb-item disabled bc-wait"><a href="#interactive-wait">Wait</a></li>
+      <li class="breadcrumb-item disabled current bc-check"><a href="#interactive-check">Check</a></li>
+    </ul><!--/.breadcrumb.prereqs-->
+  </div><!--/.breadcrumbs-wrap-->
+  <button id="get-tx-button" class="btn btn-primary connection-required"
+    title="Connection to Test Net required" disabled>Check transaction status</button>
+  <div id="get-tx-output"></div>
+</div>
+</div>
+<script type="application/javascript">
+  $("#get-tx-button").click( async function() {
+    // Wipe previous output
+    $("#get-tx-output").html("")
+
+    const txID = $("#signed-tx-hash").text()
+    const earliestLedgerVersion = parseInt($("#earliest-ledger-version").text(), 10)
+
+    const tx = await api.getTransaction(txID, {minLedgerVersion: earliestLedgerVersion})
+
+    $("#get-tx-output").html(
+      "<div><strong>Transaction result:</strong> " +
+      tx.outcome.result + "</div>" +
+      "<div><strong>Balance changes:</strong> <pre><code>" +
+      JSON.stringify(tx.outcome.balanceChanges, null, 2) +
+      "</pre></code></div>"
+    )
+  })
```

Comment:
This function doesn't put the little check mark in the breadcrumb list after "Check".

Reply:
Was wondering if someone else would catch that. ;) I noticed after deploying.